akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala

```diff
@@ -127,7 +176,44 @@ abstract class GraphStageLogic {
   /**
    * INTERNAL API
    */
-  private[stream] var interpreter: GraphInterpreter = _
+  private[this] var _interpreter: GraphInterpreter = _
+
+  /**
+   * INTENRAL API
+   */
+  private[stream] def interpreter_=(gi: GraphInterpreter) = _interpreter = gi
+
+  /**
+   * INTERNAL API
+   */
+  private[stream] def interpreter: GraphInterpreter =
```

Comment:
are there hot paths that might benefit from accessing _interpreter idirectly nstead?